core/workflow-activiti/src/main/resources/userWorkflow.bpmn20.xml

```diff
@@ -82,12 +82,47 @@ under the License.
     
     <serviceTask id="delete" name="Delete" activiti:expression="#{delete.execute(execution.processInstanceId)}"/>
     <sequenceFlow id="flow99" sourceRef="delete" targetRef="theEnd"/>
+    <!-- Recertification tasks -->
+    <userTask id="RecertificationRequest" name="Recertification Request" activiti:candidateGroups="managingDirector" activiti:formKey="recertify">
```

Comment:
I removed it. I guess we need to have more flexibility on recertifier assignment. I'll open a new ISSUE in jira with some proposal.